dbms/src/Columns/ColumnDecimal.cpp

```diff
@@ -117,6 +121,36 @@ MutableColumnPtr ColumnDecimal<T>::cloneResized(size_t size) const
     return std::move(res);
 }
 
+template <typename T>
+void ColumnDecimal<T>::insertData(const char * pos, size_t /*length*/)
+{
+    data.push_back(T());
+
+    const void * src = pos;
+    void * dst = &data.back();
+
+#if __SSE2__
+    /// prevent aligned SSE load/store.
```

Comment:
There are better solutions:
- replace non-portable `__int128_t` to `boost::int128_t`;
- figure out why `pos` is unaligned and fix it.

Reply:
pos is unaligned cause of Aggregator that doesn't care about aligment. It uses our UInt128 for any 128-bit types. It looks like there would be no profit if we replace __int128 and not replace UInt128 and UInt256.